pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -330,6 +335,13 @@ type operationExecutor struct {
 	// pendingOperations keeps track of pending attach and detach operations so
 	// multiple operations are not started on the same volume
 	pendingOperations nestedpendingoperations.NestedPendingOperations
+
+	// recorder is used to record events in the API server
+	recorder record.EventRecorder
+
+	// represents a volume that should be attached to this node and
+	// mounted to the PodName.
+	volumeToMount VolumeToMount
```

Comment:
@saad-ali - I pass that struct into other functions because it has the api.Pod object which I used for the recorder event.  See lines 589, 598, 926, etc...

Maybe I don't need to do that, but the pod is only passed into the VolumeToMount struct, so I'm just reusing that for the OE struct so i can easily reference the pod anywhere in the file??? am I way off on that?